GMG/Core/Utils/AudioLevelMeter.swift

```diff
@@ -0,0 +1,137 @@
+//  Copyright © 2025 ADA 4th GMG. All rights reserved.
+
+import AVFoundation
+import Accelerate
+import SwiftUI
+
+enum AudioLevelMeterError: Error {
+    case failedToCreateBuffer
+}
+
+enum AudioLevelMeter {
+    nonisolated static func calculateLevel(from fileURL: URL, windowPerSeconds: TimeInterval = 0.1)
+        throws -> [Float]
+    {
+        let file = try AVAudioFile(forReading: fileURL)
+
+        let foramt = file.processingFormat
+        let sampleRate = foramt.sampleRate
+
+        let framesPerWindow = AVAudioFrameCount(sampleRate * 0.1)
```

Comment:
이 부분에서 windowPerseconds 받은 값 사용하면 어떨까요?
```suggestion
        let framesPerWindow = AVAudioFrameCount(sampleRate * windowPerseconds)
```

Reply:
메서드 분리하면서 적용을 잊어버렸나보네요! 수정했습니다.